.github/pull_request_template.md

```diff
@@ -2,30 +2,40 @@
 <!-- This PR template can be edited in the [.github](https://github.com/semdatex/.github) repository. -->
 
 ## Description
+
 ### What is included in this PR?
+
 <!-- Provide a summary of the changes in this PR. -->
 
 ### Why is this PR needed?
+
 <!-- Tell us why this change is necessary. What values does it give us. -->
 
 ### Jira issues
+
 <!-- Define which jira issues are related to this PR. -->
+
 - Fixes <!-- Bug Issue number -->
 - Resolves <!-- Story Issue number -->
 
 ## Checklists
 
 ### TODOs before requesting a review:
+
 <!-- Mark all the done checkboxes with an x -->
+
 - [ ] I have provided a detailed description of this PR above.
 - [ ] I have extended all predefined checklist in this PR description to provide an exhaustive list of todos. Including a list of things that need to be tested manually, in addition to automated tests.
+- [ ] Add the task Jira link to the file change log.
```

Comment:
Couly you please add: _to german and english_ change log. Otherwise they might differ.

Reply:
I am totally lost. I would not not how to complete this TODO.

Is this something specific to ICN? If so perhaps you should edit the template specific to that project instead of the global one?